cms_delete_content/models/test_models.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, tools
+
+testing = tools.config.get('test_enable')
+
+if testing:
```

Comment:
tried, does not work AFAIR :(
Unless you import test.test_models into __init__. Which does not change that much I'd say.